twitcher/datatype.py

```diff
@@ -39,16 +35,22 @@ def type(self):
     @property
     def public(self):
         """Flag if service has public access."""
+        # TODO: public access can be set via auth parameter.
         return self.get('public', False)
 
     @property
-    def c4i(self):
-        """Flag if service is by climate4impact."""
-        return self.get('c4i', False)
+    def auth(self):
```

Comment:
Will 'c4i' need to be replaced by 'auth' in Magpie too? (https://github.com/Ouranosinc/Magpie/blob/master/providers.cfg)
I know that this value must be pushed to Phoenix until it is removed, and it is parsed in current Magpie code.

Reply:
Since we will no longer push anything to phoenix the providers.cfg will only need stuff actually required by magpie. I would say that "public" and "c4i" could be removed.